spyder/app/mainwindow.py

```diff
@@ -1240,6 +1240,7 @@ def post_visible_setup(self):
         # Show dialog with missing dependencies
         self.report_missing_dependencies()
 
+        self.menuBar().raise_()

```

Comment:
Please add a comment above this line explaining why this is needed and  mentioning the issue where the error was reported.